api/v1alpha1/contentlibraryitem_types.go

```diff
@@ -55,23 +55,35 @@ type CertificateVerificationInfo struct {
 
 // FileInfo represents the information of a file in a content library item in vCenter.
 type FileInfo struct {
-	// Name specifies the name of the file in vCenter.
 	// +required
+
+	// Name specifies the name of the file in vCenter.
 	Name string `json:"name"`
 
-	// SizeInBytes indicates the library item file size in bytes on storage in vCenter.
 	// +required
+
+	// SizeInBytes indicates the library item file size in bytes on storage in vCenter.
 	SizeInBytes resource.Quantity `json:"sizeInBytes"`
 
+	// +required
+
 	// Version indicates the version of the library item file in vCenter.
 	// This value is incremented when a new copy of the file is uploaded to vCenter.
-	// +required
 	Version string `json:"version"`
 
-	// Cached indicates if the library item file is on storage in vCenter.
 	// +required
 	// +kubebuilder:default=false
+
+	// Cached indicates if the library item file is on storage in vCenter.
 	Cached bool `json:"cached"`
+
+	// +required
```

Comment:
If a file is not cached, does CL API show the storage of the file? if not, then this needs to be optional.

Reply:
Updated it to be an optional field.